template/config/vitest/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "scripts": {
+    "test:unit": "vitest --environment jsdom"
```

Comment:
But Vitest switches to `run` automatically in CI environments…

Reply:
Yeah but I usually have a different command (for example to add code coverage) for CI. Maybe that's enough!

Reply:
Vitest coverage requires `c8` as a peer dependency. It can be easily installed in an interactive shell following a prompt from Vitest, but not that easy in CI environments.

🤔 The developer experience in Vitest is quite different and unique when compared with other testing tools.
So I think having only one script for unit testing also helps strengthen this impression.

Let's leave it as-is for now and wait for more user feedback.